frontend/components/admin/users/authorized-user.tsx

```
@@ -25,13 +31,23 @@ import { Textarea } from "@/components/ui/textarea";
 import imageCompression from "browser-image-compression";
 import { updateUserInfo } from "@/lib/requests/authorized-user";
 
+interface UserActivity {
+  timestamp: string;
+  type: "enrollment" | "page_view" | "lesson_view" | "completion" | "rating";
+  description: string;
+  details?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)